app/src/main/AndroidManifest.xml

```diff
@@ -12,7 +12,8 @@
         android:label="@string/app_name"
         android:networkSecurityConfig="@xml/network_security_config"
         android:supportsRtl="true"
-        android:theme="@style/OwnCloud">
```

Comment:
Thanks, wanted to remove all possible Nextcloud/ownCloud-References long time ago...